src/components/button-toggle/button-toggle.component.tsx

```diff
@@ -16,8 +17,8 @@ import { InputGroupContext } from "../../__internal__/input-behaviour";
 export interface ButtonToggleProps
   extends ButtonToggleInputProps,
     Partial<StyledButtonToggleLabelProps> {
-  /** A required prop. This is the button text. */
-  children: React.ReactNode;
+  /** Text to display for the button. */
+  children?: React.ReactNode;
```

Comment:
Why has this changed from required to optional? If it's been done for a reason that's fine, just not sure if it's been done by accident or not.